releases/bin/server.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:appengine/appengine.dart';
+import 'package:releases/common/handler.dart';
+import 'package:releases/common/router.dart';
+import 'package:releases/requests/github_webhook.dart';
+import 'package:releases/requests/readiness_check.dart';
+
+Future<void> main() async {
+  await withAppEngineServices(() async {
```

Comment:
Since the motivation for this library is to make a package that is externally accessible then we should add the ability to register endpoints with this router.

Reply:
This releases project won't be the common library.

My expectation is that `bin/server.dart` will still list all the server specific handlers to be added, and `Router` + `Handler` can come from `package:cocoon`

Reply:
Okay that makes sense and does create a delineation between what you are doing and what I am seeking to deliver.